src/averageAges.js

```diff
@@ -15,13 +15,21 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mens = century
+    ? people.filter((person) => person.sex === 'm'
+        && Math.ceil(person.died / 100) === century)
+    : people.filter((person) => person.sex === 'm');
+
+  return averageAge(mens);
 }
```

Comment:
```suggestion
  const men = century
    ? people.filter((person) => person.sex === 'm'
        && Math.ceil(person.died / 100) === century)
    : people.filter((person) => person.sex === 'm');

  return averageAge(men);
}
```

![image](https://user-images.githubusercontent.com/80700247/213859612-53ea1780-b64b-4c40-bf70-5c17f9fa9ae2.png)